modules/cco-ccoctl-creating-individually.adoc

```diff
@@ -0,0 +1,122 @@
+** `_<aws-region>_` is the AWS region in which cloud resources will be created.
+** `_<path_to_ccoctl_output_dir>_` is the path to the public key file that the `ccoctl create key-pair` command generated.
++
+.Example output:
++
+[source,terminal,subs="+quotes"]
+----
+2021/04/13 11:16:09 Bucket __<name_prefix>__-oidc created
+2021/04/13 11:16:10 OpenID Connect discovery document in the S3 bucket __<name_prefix>__-oidc at .well-known/openid-configuration updated
+2021/04/13 11:16:10 Reading public key
+2021/04/13 11:16:10 JSON web key set (JWKS) in the S3 bucket __<name_prefix>__-oidc at keys.json updated
+2021/04/13 11:16:18 Identity Provider created with ARN: arn:aws:iam::__<aws_account_id>__:oidc-provider/__<name_prefix>__-oidc.s3.__<aws_region>__.amazonaws.com
+----
++
+Where `02-openid-configuration` is a discovery document and `03-keys.json` is a JSON web key set file.
+// Lin's output has openid-configuration (not 02-openid-configuration) and keys.json (not 03-keys.json) - does something in this output need correction?
+// I believe based on AWS docs that the string `301721915996` in the test plan is the AWS account ID, but would like confirmation on that.
++
+This command also creates a YAML configuration file in `/_<path_to_ccoctl_output_dir>_/manifests/cluster-authentication-02-config.yaml`.
+// Does the user need to know more about this?
+
+. Create IAM roles for each component in the cluster.
+
+.. Extract the list of `CredentialsRequest` objects from the {product-title} release image:
++
+[source,terminal,subs="+quotes"]
+----
+$ oc adm release extract --credentials-requests --cloud=aws --to=./credrequests quay.io/__<path_to>__/openshift-release:__<version>__
+----
+// Can `--to=./credrequests` be replaced with `--to=<path_to_directory_with_list_of_credentials_requests>/credrequests`?
+
+.. Process each `CredentialsRequest` object in the `./credrequests` directory with `ccoctl`:
+// Does the user have to repeat this for each? OR is it getting all at once from the directory?
++
+[source,terminal,subs="+quotes"]
+----
+$ ccoctl aws create-iam-roles --name-prefix=__<name_prefix>__ --region=__<aws_region>__ --credentials-requests-dir=__<path_to_directory_with_list_of_credentials_requests>__/credrequests --identity-provider-arn=arn:aws:iam::__<aws_account_id>__:oidc-provider/__<name_prefix>__-oidc.s3.__<aws_region>__.amazonaws.com
+----
++
+[NOTE]
+====
+For AWS GovCloud, you must also specify your region with the `--region` parameter.
+====
++
+For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and a permissions policy as defined in each `CredentialsRequest` object from the {product-title} release image.
+
+.Verification
+
+You can verify that the IAM roles are created by checking the secret files in the `_<path_to_ccoctl_output_dir>_/manifests` directory.
```

Comment:
Same comment as before. To get the llist of IAM Roles you need to query AWS directly. This is just the Secrets that will be placed into the cluster.